legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/util/LogicalPlanUtils.java

```diff
@@ -215,6 +215,20 @@ public static void removeField(List<Value> fieldsList, String fieldName)
         );
     }
 
+    public static void replaceField(List<Value> fieldsList, String oldFieldName, String newFieldName)
+    {
+        fieldsList.forEach(field ->
+        {
+            if (field instanceof FieldValue && ((FieldValue) field).fieldName().equalsIgnoreCase(oldFieldName))
```

Comment:
Should we use equalsIgnoreCase or just equals?

Reply:
fixed